src/Microsoft.Azure.WebJobs.Extensions.Kafka/Diagnostics/ActivityProvider.cs

```diff
@@ -0,0 +1,62 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using Microsoft.Azure.WebJobs.Host.Executors;
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Text;
+
+namespace Microsoft.Azure.WebJobs.Extensions.Kafka.Diagnostics
+{
+    internal class ActivityProvider
+    {
+        public static readonly ActivitySource ActivitySource = new ActivitySource("Microsoft.Azure.Webjobs.Extensions.Kafka");
```

Comment:
scope is Public are we directly goigng to access in other classes ??

Reply:
Changed the access modifier to protected.